### DISCUSSION TITLE
Guidance for Policy Makers Subgroup

### DISCUSSION THREAD

**Discussion by rachaelbradley** [w3c/wcag3#6] [General]
**Summary:**
The Guidance for Policy Makers Subgroup concluded on 27 July 2023 and briefed the AG meeting.  

Please review the [presentation](https://docs.google.com/presentation/d/1aKV1IvHoqlUkJekki2STa6CrS9jiTJfy0bHukbMMXYc/edit#slide=id.g25ae623ede5_0_0) and  ([minutes](https://www.w3.org/2023/07/25-ag-minutes.html#item01)) if you did not attend. 

The initial draft document from the subgroup is at [Guidance on Adopting WCAG in Accessibility Policies](https://docs.google.com/document/d/14J2ok2hJutcsVfkqn6eoN0nyC9TKVj5vnMjeWzXizl4/edit). This is not the copy for comment. 

**Maturity Level of Content:** Placeholder
 
**Action Needed:** Please review the content and continue the conversation started in the meeting on this discussion thread. 

Initial questions include:
- What edits are needed to improve the draft?
- Is this work appropriate for the AGWG? If the answer is no, what changes to make it appropriate?
- Should a subgroup continue working on this now or wait till after we have the initial conformance model?

Content for Review and comment: 

To make detailed suggestions, please use the [Google Document for Comment](https://docs.google.com/document/d/14FXVKZI6IrK9ITCI4DHySyT0QfYrbiqQ6Jh4yi5ukyo/edit)

**Comment by mgifford** [comment]
We were just getting going! Looking forward to seeing what we can do to make more useful recommendations to policymakers.

**Comment by bruce-usab** [comment]
Noting that DOJ ADA proposed rule for [Accessibility of Web Information and Services of State and Local Government Entities](https://www.federalregister.gov/d/2023-15823) published today.  There is a 60 day comment period.

**Reply by awkawk** [reply]
Of particular note relevant to this subgroup is the section titled "Challenges of Defining and Measuring Compliance with this Rule"

> The Department is concerned that the type of compliance measures it currently uses in the ADA, such as the one used to assess compliance with the ADA Standards, may not be practical in the web or mobile app context. Public entities must ensure that newly designed and constructed State and local government facilities are in full compliance with the scoping and technical specifications in the ADA Standards unless full compliance is structurally impracticable. The ADA Standards require newly constructed State or local government buildings to be 100 percent compliant at all times with the applicable provisions, subject to limited compliance limitations. However, unlike buildings, public entities’ websites and mobile apps are dynamic and interconnected, and can contain a large amount of complex, highly technical, varied, and frequently changing content. Accordingly, the Department is concerned that a compliance measure similar to the one used in the other area where it has adopted specific technical standards may not work well for web content or mobile apps.

**Comment by mgifford** [comment]
Given that full compliance is very rare, how should governments spell out their policies? How do you put into policy the concept of progress over perfection? How close to perfection is sufficient and how much progress is necessary? Where do we address the realities of complex, dynamic sites, while at the same time not provide enough wiggle room so that people do not need to concern themselves with accessibility?

**Comment by GreggVan** [comment]
Thanks Andrew
of particular concern is the sentence
"Accordingly, the Department is concerned that a compliance measure similar to the one used in the other area where it has adopted specific technical standards may not work well for web content or mobile apps."

This sounds like someone has convinced them that there should not be any standards (that would include WCAG) used to enforce accessibility of web pages.    This is very concerning.   Not only ignoring WCAG or other standards but also - there is no hint as to how you would enforce accessibility if there is no standard to measure against.    On the other hand - this may be a call for comments/evidence to refute this approach.   Either way - I think we need to look carefully at this - and submit comments.

**Comment by awkawk** [comment]
@GreggVan I don't think that they have been convinced that there should be no standards at all. They do indicate that WCAG 2.1 Level AA is what they are targeting, but they also recognize the challenges that exist at scale. When WebAIM can do a review of 1M home pages, only using automated tools for testing (so no manual tests such as keyboard testing), and only 3.2% of the pages don't show errors, that suggests that there is either an awareness/commitment problem, or the standard is too difficult to fully meet, or both. 

What percent of the 32,000 sites with no automatically-detectable errors do you think would emerge from a manual review with no issues?

What percent of those sites would meet WCAG 2.1 AA for the first level of links from the home page? For the entire site?

I agree that WCAG 2.x should be the standard, but perfection is, as you have said, the enemy of the good.

**Comment by nitedog** [comment]
Just a reminder of the review questions from the Chairs:

> Initial questions include:
> 
>     * What edits are needed to improve the draft?
>     * Is this work appropriate for the AGWG? If the answer is no, what changes to make it appropriate?
>     * Should a subgroup continue working on this now or wait till after we have the initial conformance model?

**Comment by awkawk** [comment]
- > What edits are needed to improve the draft?

I added a comment, but I do think that we need to be more up-front about how realistic full conformance is for web sites.

- > Is this work appropriate for the AGWG? If the answer is no, what changes to make it appropriate?

Yes. The WG has an opportunity to speak to this important topic. Others will speak to regulators if the WG doesn't, and it would be beneficial to have a consensus opinion from the group.

This is important for the WG to think about as WCAG 3 is developed, but it is important immediately as policies are in the works on an ongoing basis.

- > Should a subgroup continue working on this now or wait till after we have the initial conformance model?

I think that the same thinking needs to be done in order to arrive at a good conformance model, so it makes sense to continue.

**Comment by GreggVan** [comment]
I’m not sure what you are trying to say.

Are you suggesting that because we have very low conformance to WCAG -  that we should say that conforming is not as important?   That sounds like it would only lead to less conformance. 

If you are saying there are too many provisions and people should be able to ignore some of them and still pass  
	— then I would agree - there are too many 
	- and we should decide which ones are not as important and drop them.    
		But the problem there is that we don’t think any of them are not important for some users.    
 However I disagree that authors should be able to decide for themselves which ones they won’t follow - and then still be able to pass.  
	 They are likely to choose hard ones that break access seriously for many people. 

We all know about bugs and things.   That is not the issue. 
	 All products have bugs and as long as they are fixed as discovered no only has ever had a problem with this. 
	 And no one has ever been successfully sued because they made they site accessible but had some bugs they were willing to fix as discovered.   So bugs is a red herring. 

Now if I am off target from what you were trying to say — please let me know.  But those were the takeaways I got from what you wrote.

Did you mean something different? 

Thanks 

gregg

———————————
Professor, University of Maryland, College Park
Founder and Director Emeritus , Trace R&D Center, UMD
Co-Founder Raising the Floor. http://raisingthefloor.org
The Global Public Inclusive Infrastructure (GPII) http://GPII.net
The Morphic project  https://morphic.org

**Comment by bruce-usab** [comment]
AWK wrote:
> I do think that we need to be more up-front about how realistic full conformance is for web sites.

+1 to that!

Shadi asks:
> What edits are needed to improve the draft?

I am of the opinion that the draft does not read like "Guidance for Policy Makers" but (as it notes) is instead an iteration on [Use Cases](https://w3c.github.io/silver/use-cases/) (which itself is an iteration on [Challenges with Accessibility Guidelines Conformance and Testing, and Approaches for Mitigating Them](https://www.w3.org/TR/accessibility-conformance-challenges/)) 

I would like to see further development of Use Cases (or Challenges), but I think that work could and should be decoupled from "Guidance for Policy Makers".

"Guidance for Policy Makers" can reference Use Cases (or Challenges) _even while_ the latter is evolving.

Said Guidance must include obvious stuff like:
- Don't change the definition of terms.
- Don't muck with the conformance model.

Neither of those points is in the present draft.

Shadi asks:
> Is this work appropriate for the AGWG? If the answer is no, what changes to make it appropriate?

Yes.  +1 for the reasons AWK provided.  Which are so good, I will take the liberty to repeat them!

> Others will speak to regulators if the WG doesn't, and it would be beneficial to have a consensus opinion from the group.
> ...it is important immediately as policies are in the works on an ongoing basis.

Shadi asks:
> Should a subgroup continue working on this now or wait till after we have the initial conformance model?

Yes, but it should IMHO be two subgroups.  One subgroup continues working on Use Case Challenges.  The second subgroup proceeds from the premise that (1) there is a conformance model, and (2) Use Cases Challenges are reasonably well documented and (even if still in draft) is itself something which can be referenced by the Guidance for Policy Makers.

**Reply by nitedog** [reply]
To be clear, @rachaelbradley asked these three questions on behalf of the AGWG Chairs.

**Reply by bruce-usab** [reply]
I very much appreciated the "reminder of the review questions from the Chairs."  You were clear.    "Shadi asks" was my shorthand for "Shadi reminds us that the chairs asked".   Please accept my apologies if it seemed I was putting words in your mouth.  They are good questions!

**Comment by susipalleroarguello** [comment]
* What edits are needed to improve the draft?
There were a lot of important topics that started to emerge as we continued the discussion. This topic is very relevant and it was difficult to set a proper scope and priorities in such short time. I personally think we were getting more interesting insights just before the project ended. I just think we need more time.

* Is this work appropriate for the AGWG? If the answer is no, what changes to make it appropriate?
It is as the upcoming changes will require some advice and support for professionals working on conformance and compliance.

* Should a subgroup continue working on this now or wait till after we have the initial conformance mode.
We should have a subgroup working now, so when we have the conformance mode we are advanced in the discussion and we can have already valuable insights and inputs to enrich the new work.
